### ISSUE TITLE
Warning when feeding an object to the sortable option of the GridColumn

### ISSUE THREAD

**Issue by Nassefferjeni** [telerik/kendo-react#485]
So I was trying to implement the allowUnsort for the kendo grid but I end up getting a warning when trying to feed an object containing the sort mode and allwUnsort boolean.

![image](https://user-images.githubusercontent.com/33683191/74757254-0c0cb780-5276-11ea-9de2-d82a27a0a4d7.png)

this is my codebase:

```
return (
            columns.filter(({ show, field }) => (show && field !== 'selected')).map(({ field, title, cell, width, filter, orderIndex, sortable, alwaysSort }, i) => (
                <GridColumn key={ i } field={ field } title={ title } cell={ cell } width={ width } filter={ filter } orderIndex={ orderIndex }
                            sortable={ sortable && { mode: 'single', allowUnsort: !alwaysSort } } columnMenu={ customMenuCell && ((eventProps) => (this._renderCustomColumnMenu(eventProps, parentColumns, false))) }
                            headerClassName={ customMenuCell && this._isColumnActive(field, parentDataState) ? 'active' : '' } />
            ))
        );
```
Weirdly thow the kendo grid is working as expected.

I know that I can use an object in the Grid component (not the grid column) but I want to control which columns I can allow unsort on and which columns don't

**Comment by simonssspirit** [comment]
@Xizario please take a look at this. The type of the prop is only set the boolean, but indeed setting the allowUnsort property works per column level:

https://stackblitz.com/edit/react-mpgxip?file=app/main.jsx

**Comment by Xizario** [comment]
It is bug in the Grid PropTypes.

**Comment by Xizario** [comment]
Fixed in v3.12.0
